### PR TITLE
'calculator' Example: use the correct class.

### PR DESCRIPTION
**Pull request by phmarek**

### CODE REVIEW

File: Examples/calculator.lisp [hunk 1]
```diff
@@ -30,17 +30,28 @@
...
 	(princ-to-string number)))
 
+(defvar *last-button* nil)
+
+(defmethod clim-internals::effective-gadget-background :around ((b push-button-pane))
+  (if (eq b *last-button*)
+      (clim-internals::make-rgb-color 0.9 0.2 0.2) 
+      (call-next-method)))
+
 (defun queue-number (number)
```
Comment (dkochmanski) [inline Examples/calculator.lisp]: is there a need for that in calculator example? I'm not especially keen with advertising internal interface.

File: Examples/calculator.lisp [hunk 2]
```diff
@@ -138,3 +149,6 @@
              (list seven eight divide)
              (list nine zero result))))))
   (:top-level (calculator-frame-top-level . nil)))
+
+#+nil
+(calculator)
```
Comment (dkochmanski) [inline Examples/calculator.lisp]: clim examples are meant to be started from `clim-demo:demodemo`, also I don't see point in adding commented out function call.

if there is some reason to have it, please use `#+(or)`, not `#+nil`

File: Examples/calculator.lisp [hunk 3]
```diff
@@ -30,17 +30,28 @@
...
+
 (defun queue-number (number)
   (lambda (gadget)
-    (declare (ignore gadget))
+    (let ((last *last-button*))
+      (if last
+          (climi::dispatch-repaint last +everywhere+)))
+    (setf *last-button* gadget)
+    (climi::dispatch-repaint gadget +everywhere+)
     (with-slots (calc-state) *application-frame*
```
Comment (dkochmanski) [inline Examples/calculator.lisp]: how custom repaint is relevant to this particular example?

File: Examples/calculator.lisp (hunk 2 above)
Comment (phmarek) [inline Examples/calculator.lisp]: Oh, sorry.

I only meant for the first commit to be in the PR.... the other was just to show what I had to do to get a button colored.

Sorry about the confusion!

**Comment by dkochmanski** [comment]
so should I close this PR, or you will remove two subsequent commits?

**Comment by phmarek** [comment]
Please kill it, I'll resubmit.... seems easier.

Thanks a lot!

**Pull request closed without merging**